services/new_mac_setup.md

```diff
@@ -0,0 +1,60 @@
+# New Mac Setup
+This is a get started guide to make it easier to setup your mac.
+
+When you get a new mac here are some helpful guides to setting it up for software development in RDSS.
+
+Use the URL below to customize/setup your unix prompt.
+We use zsh as a unix prompt. The last two setup steps(8,9) are not necessary for mac setup but will add more custimization options if you feel the need to install.
+
+https://www.freecodecamp.org/news/how-to-configure-your-macos-terminal-with-zsh-like-a-pro-c0ab3f3c1156/ 
+
+
+A new SSH key will also need to be installed on your mac. Below is the link with the steps for connecting to github using SSH keys. The SSH key will be needed to connect with your github account and grants you the ability to start adding data into PUL repositories.  
+
+https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjJ8fzNxtr5AhXokmoFHU2CC5YQFnoECAUQAQ&url=https%3A%2F%2Fdocs.github.com%2Fen%2Fauthentication%2Fconnecting-to-github-with-ssh&usg=AOvVaw0ObLt130kJ7y4IwQ-069jU
+
+New fellows may also want to rename the computer name,hostname and local hostname on your Mac machine. Below is the link to help rename.
+
+https://jumpcloud.com/blog/how-to-rename-computer-name-hostname-local-hostname-macos
+
+
+New fellows will need to download and install microsoft visual studio (VS Code) for mac os. You will also need to install the commands that allow you to launch VS Code from the iterm2 terminal command line. Link to instructions are below.
+
+https://code.visualstudio.com/docs/setup/mac
+
+
+go to the pdc_describe github page and checkout the projects dependencies and the local development setup information to become aware of the specific software plugins you will need to work on the project
```

Comment:
Remove reference to PDC describe

Add a list of applications to install with homebrew.

```
brew install gpg 
brew install postgres
brew install --cask lando
brew install asdf
```

Then, add asdf plugins.